### DISCUSSION TITLE
실행횟수를 한정된 시간내에서 테스트 해볼 수는 없나요??

### DISCUSSION THREAD

**Discussion by wonjunchoi-arc** [naver/ngrinder#872] [Q&A]
한명의 유저가 초당 몇번의 작업을 했을 때 부터 서버의 반응이 느려지는지를 테스트 하고 싶은데 
테스트기간 , 실행횟수 라는 것 중 1가지 옵션밖에 설정이 되질 않습니다.
혹시 방법이 있을까요??? 
<img width="922" alt="image" src="https://user-images.githubusercontent.com/84310920/161937550-9fd0e547-46f0-4b11-889f-fcf1db94b7e0.png">

**Comment by donggyu04** [comment]
@wonjunchoi-arc 

두 가지 실행 종료 옵션을 같이 적용할 순 없습니다.
초당 호출하는 API 횟수를 조절하고 싶으시면 스크립트의 적절한 곳에 `grinder.sleep(200)`(단위는 ms)와 같은 코드를 삽입하여 호출량을 조절하실 수 있습니다. 호출량을 적절히 조절해서 테스트 해보시기 바랍니다.

> `초당 몇번의 작업을 했을 때 부터 서버의 반응이 느려지는지` 는 호출량을 적절히 조절 했을때 TPS 변화로 예측이 가능하겠네요
